### PR TITLE
Wifi

### PR DESCRIPTION
**Pull request by Julius0530**

### CODE REVIEW

File: utils/wifi.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+    :param password: gives the password from the wifi to the pico
+    :return: ip_addresses: gives the ip addresses from the pico w
+    """
     wlan = network.WLAN(network.STA_IF)
     wlan.active(True)
     wlan.connect(ssid, password)
     while not wlan.isconnected():
         sleep(1)
         print("Connecting to WiFi...")
+    return wlan.ifconfig()
```
Comment (rmoesbergen) [inline utils/wifi.py:17]:
```suggestion
    return wlan.ifconfig()[0]
```

File: utils/wifi.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
-import network
 from time import sleep
+import network
 
-def connect(ssid, password):
+def connect(ssid: str, password: str) -> tuple[str]:
```
Comment (rmoesbergen) [inline utils/wifi.py:4]:
```suggestion
def connect(ssid: str, password: str) -> str:
```

File: utils/wifi.py [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
+    """
+    Connects the pico to the wifi
+    :param ssid: gives the wifi name to the pico
+    :param password: gives the password from the wifi to the pico
+    :return: ip_addresses: gives the ip addresses from the pico w
```
Comment (rmoesbergen) [inline utils/wifi.py:9]:
```suggestion
    :return: ip_address: returns the ip address from the pico w
```

**Review by koen1711 — CHANGES_REQUESTED** [review]
I agree with Ronald, this is uncomplete

**Review by koen1711 — COMMENTED** [review]
seems good

**Review by koen1711 — APPROVED** [review]

**Pull request merged**